### DISCUSSION TITLE
How to cancel certain shortcut keys? For example: Ctrl+r

### DISCUSSION THREAD

**Discussion by ghost** [marlonrichert/zsh-autocomplete#671] [Q&A]
### eg title
I use [atuin](https://github.com/atuinsh/atuin) for history.
This conflicts with zsh-autocomplete's `ctrl+r`
current solution is to move the atuin environment variable to the top of the list `.zshrc`
#### Any other solutions? 😦
For example, disabling zsh-autocomplete's history logging

### Make Tab go straight to the menu and cycle there
[make-tab-go-straight-to-the-menu-and-cycle-there](https://github.com/marlonrichert/zsh-autocomplete#make-tab-go-straight-to-the-menu-and-cycle-there)
```
bindkey '\t' menu-select "$terminfo[kcbt]" menu-select
bindkey -M menuselect '\t' menu-complete "$terminfo[kcbt]" reverse-menu-complete
```
This is a favorite feature, thanks a lot!
But when I use [atuin](https://github.com/atuinsh/atuin) often it doesn't work.
I had to use `source ~/.zshrc` to get back to normal.

**Comment by marlonrichert** [comment]
The order in which you source plugins matters, because they will override each others' keybindings. So, if you prefer Atuin's over Autocomplete's, then source Autocomplete first and then Atuin.
